tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs

```diff
@@ -68,5 +68,128 @@ public async Task WordPress51(string phpVersion)
+            var volume = DockerVolume.CreateMirror(hostDir);
+            var appDir = volume.ContainerDir;
+
+            // build-script to download wordpress cli and build
+            var buildScript = new ShellScriptBuilder()
+                .AddCommand($"cd {appDir}")
+                .AddCommand("curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar")
+                .AddCommand("chmod +x wp-cli.phar")
+                .AddCommand("./wp-cli.phar core download")
+                .AddCommand($"oryx build {appDir} --platform {PhpConstants.PlatformName} --language-version {phpVersion}")
+                .AddDirectoryExistsCheck("wp-admin")
+                .AddFileDoesNotExistCheck("wp-config.php")
+                .ToString();
+
+            // run script to finish wordpress configuration and run the app
+            var runScript = new ShellScriptBuilder()
+                .AddCommand($"cd {appDir}")
+                .AddFileExistsCheck("oryx-manifest.toml")
+                .AddFileExistsCheck("wp-cli.phar")
+                .AddFileExistsCheck("create_wordpress_db.sh")
+                .AddFileExistsCheck("configure_wordpress.sh")
+                .AddCommand($"chmod +x create_wordpress_db.sh && ./create_wordpress_db.sh")
+                .AddCommand($"chmod +x configure_wordpress.sh && ./configure_wordpress.sh")
+                .AddCommand($"oryx create-script -appPath {appDir} -bindPort {ContainerPort} -output {RunScriptPath}")
+                .AddCommand(RunScriptPath)
+                .ToString();
+
+            // Act & Assert
+            await EndToEndTestHelper.BuildRunAndAssertAppAsync(
+                appName, _output, volume,
+                "/bin/sh", new[] { "-c", buildScript },
+                _imageHelper.GetRuntimeImage("php", phpVersion),
+                ContainerPort,
+                "/bin/sh", new[] { "-c", runScript },
+                async (hostPort) =>
+                {
+                    // this is to test wordpress and mysql connection is working
+                    var testdbconnectiondata = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/testdb.php");
+                    Assert.DoesNotContain("Unable to connect to MySQL",testdbconnectiondata);
+                    Assert.Contains("Connected successfully", testdbconnectiondata);
+
+                    // this is to test regular wordpress site is working
+                    var data = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/wp-login.php");
+                    Assert.Contains("Powered by WordPress", data);
+                    Assert.Contains("Remember Me", data);
+                    Assert.Contains("Lost your password?", data);
+                    Assert.Contains("Back to localsite", data);
+                });
+        }
+
+        [Theory]
+        [InlineData("7.4")]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
```

Comment:
Maybe add 7.0, 5.6 version as well?

Reply:
we dont support php fpm images of 7.0 and 5.6. those are only supported in regular php images that has apache